src/Compilers/CSharp/Test/Symbol/Symbols/Source/FieldTests.cs

```diff
@@ -492,15 +492,16 @@ unsafe struct S
     private fixed byte goo[10];
 }
 ";
-            var comp = CreateEmptyCompilation(text);
+            var comp = CreateCompilation(text);
```

Comment:
📝 With an empty compilation, `System.Int32` was missing, so `FixedSize` would return 0 because the constant value was "bad".